constraints.txt

```diff
@@ -92,7 +92,7 @@ pytest==4.1.1
 python-dateutil==2.7.5
 pytoml==0.1.19
 pytz==2018.5
-raiden-contracts==0.11.0
+raiden-contracts==0.10.1
```

Comment:
My recommendation is `0.11.0` rather than `0.10.1`.  The difference
* 0.11.0 uses a dummy service token for service fees
* 0.10.1 uses WETH for service fees (which is confusing, according to Stefan).